doc/devel/min_dep_policy.rst

```diff
@@ -51,6 +51,13 @@ versions no longer support our minimum NumPy or Python.
 
 We will work around bugs in our dependencies when practical.
 
+IPython and Matplotlib do not technically depend on each other, however there
```

Comment:
I was briefly confused why this is in "Python dependencies" (reading the dependency on Python). Can we rename the section "Python dependencies" -> "Other Python packages"

The current toc structures is

- Dependency version policy
  - Python and NumPy
  - Python dependencies

And I believe "Other Python packages" is more clear.